debugger/pyproject.toml

```
@@ -22,9 +22,12 @@ pdb = ["remote-pdb>=2.1.0,<3.0.0"]
 debugpy = ["debugpy>=1.8.17,<2.0.0"]
 
 [dependency-groups]
-dev = [
-    "pytest == 8.4.1",
-    "coverage[toml] == 7.10.2",
+test = [
+    "coverage[toml] == 7.11.3",
+    "coverage-conditional-plugin == 0.9.0",
+    "httpx-retries == 0.4.5",
+    "pytest == 9.0.1",
+    "pytest-xdist == 3.8.0",
```

This set can be a little more conservative - debugger's tests don't need to do HTTP lookups, and don't have conditional testing requirements.
```suggestion
    "coverage[toml] == 7.11.3",
    "pytest == 9.0.1",
    "pytest-xdist == 3.8.0",
```